docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,366 @@
+Stable patch format
+===================
+
+Every Ubuntu LTS release during standard security maintenance period (see
+`The Ubuntu lifecycle and release cadence`_) welcomes contributions from
+anyone. However, patches must comply with the required format.
+
+.. _The Ubuntu lifecycle and release cadence: https://ubuntu.com/about/release-cycle
+
+Prerequisites
+-------------
+
+Please subscribe `here
```

Comment:
Hi @lkpdn , we can skip the "please" :)
I know it sounds polite but unnecessary.

Reply:
Hm, ok. I didn't feel so tbh (ref. https://wiki.ubuntu.com/Kernel/Dev/StablePatchFormat) but will fix. Thanks!